code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -90,10 +90,11 @@
 	name = "Tail lash"
 	icon_icon = 'icons/effects/effects.dmi'
 	button_icon_state = "tail"
+	check_flags = AB_CHECK_LYING | AB_CHECK_CONSCIOUS | AB_CHECK_STUNNED
 
 /datum/action/innate/tail_lash/Activate()
 	var/mob/living/carbon/human/user = owner
-	if(!user.restrained() || !user.buckled)
+	if((user.restrained() && user.pulledby) || user.buckled)
```

Comment:
why not just check for `user.incapacitated()`?

Reply:
The check for other incapacitations shouldn't be needed, as the check_flags should handle everything else.